boards/arm/mps2_an521/Kconfig.board

```diff
@@ -1,8 +1,20 @@
 # Copyright (c) 2018-2019 Linaro Limited
 # SPDX-License-Identifier: Apache-2.0
 
-config BOARD_MPS2_AN521
-	bool "ARM Cortex-M33 SMM on V2M-MPS2 (AN521)"
-	depends on SOC_MPS2_AN521
+config BOARD_MPS2_AN521_CPU0
+	bool "ARM Cortex-M33 SMM on V2M-MPS2 (AN521) (CPU0)"
+	depends on SOC_MPS2_AN521_CPU0
+	select QEMU_TARGET
+	select HAS_COVERAGE_SUPPORT
+
+config BOARD_MPS2_AN521_CPU0NS
```

Comment:
also a nit, but this feels a bit unnatural to me, and `BOARD_MPS2_AN521_CPU0_NS` feels easier to read, and is more consistent with current `_ns` or `_nonsecure` naming conventions?

Reply:
the Nordic boards have `ns` and `NS` without the underscores, so we lack consistency. I 'll change this, anyways, to what you suggest. But i think we need to agree on some rules here for the Non-Secure variant naming.

Reply:
Fixed.